.github/workflows/format.yml

```diff
@@ -14,23 +14,22 @@ jobs:
     name: Format (Python)
     runs-on: ubuntu-22.04
     steps:
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.11'
 
-      - name: Install dependencies
-        run:
-            python -m pip install click==8.0.4 black==23.7.0 isort==5.13.2
+      - uses: actions/checkout@v3
 
-      - name: Checkout PennyLane-Lightning
-        uses: actions/checkout@v3
-
-      - name: Run isort
-        run: python -m isort --profile black ./pennylane_lightning/ ./mpitests ./tests --check --diff
+      - name: python-isort
+        uses: isort/isort-action@v1.1.0
+        with:
+          configuration: --py 311 --profile black -l 100 -p pennylane_lightning --check --diff
+          isort-version: 5.13.2
+          requirements-files: "requirements-dev.txt"
+          sort-paths: "./pennylane_lightning/ ./mpitests ./tests"
 
-      - name: Run Black
-        run: python -m black -l 100 pennylane_lightning/ tests/ --check --verbose
+      - uses: psf/black@stable
+        with:
+          options: "-l 100 --check --verbose"
+          src: "./pennylane_lightning/ ./mpitests ./tests"
+          version: "23.7.0"      
```

Comment:
We usually avoid third-party actions, but I guess we can rely on that.

Reply:
Yes, but they are the official isort/black collab actions so I thought it would be ok.

Reply:
No problems for me.